eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServer.java

```diff
@@ -206,4 +206,17 @@ public QueryResponse checkVariantExists(@PathVariable("variantId") String varian
         }
     }
 
+    @RequestMapping(value = "/count", method = RequestMethod.GET)
+    public QueryResponse countTotalNumberOfVariants() {
```

Comment:
Now I'm thinking that why not just `countVariants`? It basically expresses the same idea.